corehq/apps/hqadmin/templates/hqadmin/raw_couch.html

```diff
@@ -0,0 +1,34 @@
+{% load hq_shared_tags %}{% load compress %}
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Doc {{ doc_id }}</title>
+        {% compress css %}
+        <link type="text/less" rel="stylesheet" media="all" href="{% static 'style/less/hqstyle-core.less' %}"/>
+        {% endcompress %}
+        {% if less_debug %}
+            <script src="{% static 'style/lib/lessjs/less-1.3.1.min.js' %}"></script>
```

Comment:
Do you know we distribute any (compiled) bootstrap css?  That's all I want here, so I copypasta'd this from elsewhere, as it appears to be the only way to get bootstrap styling.  It doesn't even have to be HQ style